modules/extensions/src/brushing/brushing-extension.ts

```diff
@@ -101,4 +101,17 @@ export default class BrushingExtension extends LayerExtension {
       });
     }
   }
+
+  draw(this: Layer<BrushingExtensionProps>, params: any, extension: this) {
+    const {viewport, mousePosition} = params.context;
+    const {brushingEnabled, brushingRadius, brushingTarget} = params.moduleParameters;
+    const brushingProps: BrushingModuleSettings = {
+      viewport,
+      mousePosition,
+      brushingEnabled,
+      brushingRadius,
+      brushingTarget
+    };
+    this.setShaderModuleProps({brushing: brushingProps});
```

Comment:
Again, the function is called `set...Props` but it takes `...Settings`

In luma I have been trying to standardize on `...Props` for shader modules.

Reply:
Renamed across all extensions